Neos.Neos/Classes/Fusion/Cache/GraphProjectorCatchUpHookForCacheFlushing.php

```diff
@@ -269,4 +275,9 @@ public function onAfterCatchUp(): void
         }
         $this->flushWorkspaceRequestsOnAfterCatchUp = [];
     }
+
+    private function hasMatchingFlushWorkspaceRequest(FlushNodeAggregateRequest $request): bool
```

Comment:
the method name is not really understandable to me at first sight ^^ but just a minor nitpick

Reply:
What is your suggestion?

Reply:
i just inlined it because i think it just adds complexity.
For instance one has to be careful to call this method before `flushWorkspaceRequestsOnAfterCatchUp = []` is done for example.
Inlined with the nice commend its much better understood and also statically analyseable